com.autodesk.Eagle.appdata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>com.autodesk.Eagle</id>
+  <name>Autodesk Eagle</name>
+  <summary>Powerful suite for schematic capture and printed circuit board design (aka eaglecad)</summary>
+  <description>
```

Comment:
The description should contain a disclaimer that it's not an officially maintained app (see commercial Jetbrains applications) and mention it requires an account to work.